var/spack/repos/builtin/packages/py-black/package.py

```diff
@@ -17,14 +17,15 @@ class PyBlack(PythonPackage):
 
     maintainers = ["adamjstewart"]
 
+    version("22.12.0", sha256="229351e5a18ca30f447bf724d007f890f97e13af070bb6ad4c0a441cd7596a2f")
     version("22.10.0", sha256="f513588da599943e0cde4e32cc9879e825d58720d6557062d1098c5ad80080e1")
     version("22.8.0", sha256="792f7eb540ba9a17e8656538701d3eb1afcb134e3b45b71f20b25c77a8db7e6e")
     version("22.6.0", sha256="6c6d39e28aed379aec40da1c65434c77d75e65bb59a1e1c283de545fb4e7c6c9")
     version("22.3.0", sha256="35020b8886c022ced9282b51b5a875b6d1ab0c387b31a065b84db7c33085ca79")
     version("22.1.0", sha256="a7c0192d35635f6fc1174be575cb7915e92e5dd629ee79fdaf0dcfa41a80afb5")
 
     # This is the last v21 release, and it's needed to format for Python 2.7
-    version("21.12b0", sha256="77b80f693a569e2e527958459634f18df9b0ba2625ba4e0c2d5da5be42e6f2b3")
+    version("21.12b0", sha256="77b80f693a569e2e527958459634f18df9b0ba2625ba4e0c2d5da5be42e6f2b3", deprecated=True)
```

Comment:
@tgamblin @alalazo this version is no longer needed, correct? Just want to make sure I don't break bootstrapping or style tests again.

Reply:
@adamjstewart: yep this is fine -- black and other bootstrap specs are here: https://github.com/spack/spack/blob/9f0bb4301f0514a9367ca9289ceb6a5cf2b3bfa0/lib/spack/spack/bootstrap/environment.py#L162-L184.  This isn't required.